### PR TITLE
[BUGFIX] Resolved known issues

### PR DESCRIPTION
**Pull request by techtocore**
Signed-off-by: Akash Ravi <akashkravi@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,24 @@
 FROM gitlab-registry.cern.ch/swan/docker-images/dev-jupyter
+
+LABEL maintainer="Akash Ravi <akashkravi@gmail.com>"
```
Comment (diocas) [inline Dockerfile:3]: Instead of `LABEL`, you can user `MAINTAINER`

File: Dockerfile [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
 RUN cd jupyter && \
 rm -rf /opt/conda/lib/python3.7/site-packages/notebook/templates && \
 cp -rl templates /opt/conda/lib/python3.7/site-packages/notebook/ && \
 ls && \
 cd SwanContents && pip install --no-deps . && \
 jupyter nbextension install --py --system swancontents
+
+RUN git clone https://github.com/Anaconda-Platform/nb_conda_kernels.git
+RUN cd nb_conda_kernels && \
+cat requirements.txt | xargs -n 1 pip install || true && \
```
Comment (diocas) [inline Dockerfile:17]: `pip install -r requirements.txt`

Comment (diocas) [reply Dockerfile:17]: Can't you just `conda/pip install nb_conda_kernels`?

**Review by diocas — CHANGES_REQUESTED** [review]
Small things

File: Dockerfile (hunk 1 above)
Comment (techtocore) [reply Dockerfile:3]: I found this method on https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

File: Dockerfile (hunk 2 above)
Comment (techtocore) [reply Dockerfile:17]: > `pip install -r requirements.txt`

This is what I had in the beginning. Unfortunately, this breaks even if any one of the packages fails to install. I had a situation where it caused an issue. However, if the current version works, I'll revert it back to  `pip install -r requirements.txt`

File: Dockerfile (hunk 1 above)
Comment (diocas) [reply Dockerfile:3]: And now I know :)

Comment (diocas) [reply Dockerfile:3]: Thanks

**Pull request merged**